### PR TITLE
Short demote all

### PR DESCRIPTION
**Pull request by roironn**

### CODE REVIEW

File: packages/core/test/workspace/local/workspace.test.ts [hunk 1]
```diff
@@ -191,4 +196,53 @@ describe('local workspace', () => {
...
+  describe('demoteAll', () => {
+    beforeAll(() => {
+      mockExists.mockResolvedValue(true)
+      const mockLoad = ws.loadWorkspace as jest.Mock
+      mockLoad.mockResolvedValue({
+        demoteAll: jest.fn(),
+      })
+    })
+
+    describe('with only one env', () => {
```
Comment (adamro) [inline packages/core/test/workspace/local/workspace.test.ts:209]: maybe also add a test to the case of 2 envs, that even if they are empty it should use the multienvSource demoteAll func

**Review by adamro — APPROVED** [review]

**Review by adamro — APPROVED** [review]
